test/components/DayPicker_spec.jsx

```diff
@@ -260,6 +261,20 @@ describe('DayPicker', () => {
       });
     });
 
+    describe('#componentWillReceiveProps', () => {
```

Comment:
Yeah, at Airbnb we use jsdom afaict and `mocha-wrap` (https://github.com/ljharb/mocha-wrap) plugin, but I think there was some discussion on what we plan to do for our open-source repos. @ljharb do you have thoughts on getting enzym's `mount` to work in this repo? I haven't really thought about it, but we should really open an issue I guess.